index.html

```diff
@@ -43,6 +44,188 @@ <h3 class="motto" >
+  <h1>
+    Multi-Post<br>Stories
+  </h1>
+  <div class="canopy-backend">
+  <ul>
+  <li class="canopy">CANOPY</li>
+  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="back-end">Back End Dev</li>
+  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="date">2015</li>
+  </ul>
+</div>
+<p class="daily-read">A daily selection of privately <br>personalized reads; no accounts or<br> sign-ups required.</p>
+
+<div class="dev-lang">
+<ul>
+  <li class="lang">HTML</li>
+  <li class="lang">CSS</li>
+  <li class="lang">JavaScript</li>
+
+</ul>
+</div>
+<h3 class="see-pr">See project</h3>
+
+</div>
+
+</section>
+<section class="about-myself">
+<h1 class="me">
+  About Myself
+</h1>
+<p class="hello">
+  Hello I’m a software developer! I can<br> help you build a product , feature or <br>website. Look through some of my work <br> and experience! If you like what you see <br>and have a project you need coded,<br> don’t hesitate to contact me.
+</p>
+  <h3 class="motto">
+    LET’S CONNECT
+  </h3>
+  <div class="social-media">
+    <ul>
+        <li><a ><img class="twitter icons"src="images\Vector.svg" alt="twitter"></a></li>
+        <li><a><img class="linkedin icons" src="images\Linkedin icon@2x (1).png" alt="linkedin"></a></li>
+        <li><a><img class="vector-1 icons" src="images\Vector (1).svg" alt=""></a></li>
+        <li><a><img class="vector-2 icons" src="images\Vector (2).svg" alt=""><a></a></li>
+        <li><a><img class="vector-3 icons" src="images\Vector (3).svg" alt=""><a></li>
+    </ul>              
+   
+  </div>
+  <h3 class="resume">
+    Get my resume
+  </h3>
+  <ul>
+    <li class="lang-drop">
+      <h2 class="lg">Languages</h2>
+      <i  class="fa">&#xf107;</i>
+    </li>
+    <li class="js">
+      <img class="icon" src="images\Ellipse 1.png" alt="">
+      <h5 class="txt">JavaScript</h5>
+    </li>
+    <li class="js">
+      <img class="icon" src="images\Ellipse 1 (1).png" alt="">
+      <h5 class="txt">HTML</h5>
+    </li>
+    <li class="js">
+      <img class="icon"src="images\Ellipse 1 (2).png" alt="">
+      <h5 class="txt">CSS</h5>
+    </li>
+    
+    <li class="side-drop">
+      <h2 class="frame">Frameworks</h2>
+    <i  class="fa">&#xf105;</i>
+    </li>   
+    
+    <li class="side-drop">
+      <h2 class="skill">Skills</h2>
+      <i  class="fa">&#xf105;</i>
+    </li>
+    
+
+  </ul>
+</section>
+<section class="contact-me">
+  <h1 class="contact">
+    Contact me
+  </h1>
+  <p>
+    If you have an application you <br>are interested in developing, a <br>feature that you need built or a <br>project that needs coding.<br> 
+I’d love to help with it!
+  </p>
+  <h3 class="ipsum">Lorem Ipsum</h3>
+  <h3 class="ipsum">loremipsum@gmail.com</h3>
+  <form action="">    
+    <input class="input" type="text" id="message" name="message" placeholder="Write your message here...">
+    <button class="get-in"type="submit">Get in touch</button>
+  </form>
+
+</section>
```

Comment:
Please remove the contact section as it's part of the project requirement in this milestone. You will be working on that in the coming week.